web/src/components/FilingsCalendar.test.tsx

```diff
@@ -649,4 +649,58 @@ describe("<FilingsCalendar />", () => {
       expect(screen.queryByText(Config.dashboardDefaults.calendarGridViewButton)).not.toBeInTheDocument();
     });
   });
+
+  describe("calendar list view, view more functionality", () => {
+    function renderCalendarWithEntries(numberOfCalendarEntries: number) {
```

Comment:
love this helper :) we usually use arrow functions pretty much everywhere, for no particular reason, but it might be nice here also just to keep style consistency